rocketbelt/components/images/_images-lazyload.scss

```diff
@@ -46,7 +46,15 @@
 
     &.lazyload-small {
       opacity: 0.5;
-      transition: opacity 200ms ease-in-out;
+      transition: opacity 200ms ease-in-out;     
+    }
+    
+    &.errored{
```

Comment:
It's small, but can you please add a space between the selector & brace here and in the nested selector to pass the project's linting rules? (If you want I can walk you through setting up `sass-lint` in VS to catch this stuff earlier.)

Reply:
yes i will update this. i usually do have that space, oops